views/ipv/page/pyi-triage-another-way.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% set pageTitleKey = 'pages.pyiTriageAnotherWay.title' %}
+{% set googleTagManagerPageId = "pyiTriageAnotherWay" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.pyiTriageAnotherWay.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pyiTriageAnotherWay.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#pyiTriageAnotherWay" %}
+
+{% block content %}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pyiTriageAnotherWay.header' | translate }}</h1>
+  <p class="govuk-body">{{ 'pages.pyiTriageAnotherWay.content.paragraph1' | translate }}</p>
+  <h2 class="govuk-heading-m">
+    {{ 'pages.pyiTriageAnotherWay.content.subHeading1' | translate }}
+  </h2>
+
+  <form id="pyiTriageAnotherWayForm" action="/ipv/page/{{ pageId }}" method="POST">
+    <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+    {% set radiosConfig = {
+      idPrefix: "journey",
+      name: "journey",
+      items: [
+        {
+          value: "returnToRp",
+          text: 'pages.pyiTriageAnotherWay.content.formRadioButtons.continueToServiceButtonText' | translate
+        },
+        {
+          value: "back",
```

Comment:
It would only automatically use the back functionality if we dont specifically define back step. These actions are going back to previous page so does make sense to be called back. The existing buffer page also uses the keyword back so keep consistency with that. Happy to change it if you still think we should?

Reply:
After more thought, if we decide to add a back button to these pages its best to avoid this now, I will make the change